src/coreclr/src/ildasm/dasm_sz.cpp

```diff
@@ -211,6 +211,7 @@ unsigned SizeOfField(PCCOR_SIGNATURE *ppSig, ULONG cSig, IMDInternalImport* pImp
                     case ELEMENT_TYPE_CMOD_OPT  :
                     case ELEMENT_TYPE_CMOD_REQD :
                         *ppSig += CorSigUncompressToken(*ppSig, &tk);
+                         __fallthrough; // TODO: is this correct?
```

Comment:
@davidwrighton I am not sure if this fallthrough here is intentional, can you please advise me on this one?

Reply:
Yes, this looks intentional. I don't think its well written, but intentional, and not functionally wrong... yes.